plugins/processors/awsappsignals/config/resolvers.go

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package config
+
+const (
+	// PlatformGeneric Platforms other than Amazon EKS
+	PlatformGeneric = "generic"
+	// PlatformEKS Amazon EKS platform
+	PlatformEKS = "eks"
+)
+
+type Resolver struct {
+	ClusterName string `mapstructure:"cluster_name"`
```

Comment:
Just thinking ahead, would it be better to call this `Name` or `UID` instead of `ClusterName`? If we want to start using this for generic platform at a later point.

Reply:
Updated to `Name`.